storage/store_metrics.go

```diff
@@ -74,9 +75,95 @@ var (
+		Help: "Nanoseconds spent in store.processRaft() processing replica.Tick()"}
+
+	// Raft message metrics.
+	metaRaftRcvdMessages = map[raftpb.MessageType]metric.Metadata{
+		raftpb.MsgHup: {
+			Name: "raft-messages.rcvd.MsgHup",
+			Help: "Total number of MsgHup messages received by this store",
+		},
+		raftpb.MsgBeat: {
+			Name: "raft-messages.rcvd.MsgBeat",
+			Help: "Total number of MsgBeat messages received by this store",
+		},
+		raftpb.MsgProp: {
+			Name: "raft-messages.rcvd.MsgProp",
+			Help: "Total number of MsgProp messages received by this store",
+		},
+		raftpb.MsgApp: {
+			Name: "raft-messages.rcvd.MsgApp",
+			Help: "Total number of MsgApp messages received by this store",
+		},
+		raftpb.MsgAppResp: {
+			Name: "raft-messages.rcvd.MsgAppResp",
+			Help: "Total number of MsgAppResp messages received by this store",
+		},
+		raftpb.MsgVote: {
+			Name: "raft-messages.rcvd.MsgVote",
+			Help: "Total number of MsgVote messages received by this store",
+		},
+		raftpb.MsgVoteResp: {
+			Name: "raft-messages.rcvd.MsgVoteResp",
+			Help: "Total number of MsgVoteResp messages received by this store",
+		},
+		raftpb.MsgSnap: {
+			Name: "raft-messages.rcvd.MsgSnap",
+			Help: "Total number of MsgSnap messages received by this store",
+		},
+		raftpb.MsgHeartbeat: {
+			Name: "raft-messages.rcvd.MsgHeartbeat",
+			Help: "Total number of MsgHeartbeat messages received by this store",
+		},
+		raftpb.MsgHeartbeatResp: {
+			Name: "raft-messages.rcvd.MsgHeartbeatResp",
+			Help: "Total number of MsgHeartbeatResp messages received by this store",
+		},
+		raftpb.MsgUnreachable: {
+			Name: "raft-messages.rcvd.MsgUnreachable",
+			Help: "Total number of MsgUnreachable messages received by this store",
+		},
+		raftpb.MsgSnapStatus: {
+			Name: "raft-messages.rcvd.MsgSnapStatus",
+			Help: "Total number of MsgSnapStatus messages received by this store",
+		},
+		raftpb.MsgCheckQuorum: {
+			Name: "raft-messages.rcvd.MsgCheckQuorum",
+			Help: "Total number of MsgCheckQuorum messages received by this store",
+		},
+		raftpb.MsgTransferLeader: {
+			Name: "raft-messages.rcvd.MsgTransferLeader",
+			Help: "Total number of MsgTransferLeader messages received by this store",
+		},
+		raftpb.MsgTimeoutNow: {
+			Name: "raft-messages.rcvd.MsgTimeoutNow",
+			Help: "Total number of MsgTimeoutNow messages received by this store",
+		},
+	}
+
+	metaRaftSentMessages = metric.Metadata{
+		Name: "raft-messages.sent.total",
+		Help: "Total number of outgoing messages queued from this store (including messages that were later dropped)",
+	}
+
+	metaRaftDroppedMessages = metric.Metadata{
+		Name: "raft-messages.sent.dropped",
+		Help: "Total number of outgoing messages from this store that were dropped by the transport",
+	}
+
+	metaRaftTicks = metric.Metadata{Name: "raft.ticks",
+		Help: "Total number of Raft ticks processed"}
+
+	metaRaftTransportQueueSize = metric.Metadata{Name: "raft.transportqueue.size",
```

Comment:
is this for messages being sent only? If so, I would go with `raft.sent.pending`. Otherwise, maybe plain `raft.pending`.
Putting `size` in the name makes it sounds like capacity rather than current usage.